datafusion/sqllogictest/test_files/subquery.slt

```diff
@@ -433,17 +433,32 @@ logical_plan
 08)----------TableScan: t1 projection=[t1_int]
 
 #invalid_scalar_subquery
-statement error DataFusion error: check_analyzed_plan\ncaused by\nError during planning: Scalar subquery should only return one column, but found 2: t2.t2_id, t2.t2_name
+statement error
 SELECT t1_id, t1_name, t1_int, (select t2_id, t2_name FROM t2 WHERE t2.t2_id = t1.t1_int) FROM t1
+----
+DataFusion error: Invalid (non-executable) plan after Analyzer
+caused by
+Error during planning: Scalar subquery should only return one column, but found 2: t2.t2_id, t2.t2_name
```

Comment:
This change is was introduced with `cargo test --test sqllogictests -- --complete`, passing locally on macos -- but fails in CI. I'll figure out the whys later.

Reply:
sqllogictest's behavior with multi-line errors is strange. I haven't debugged it, but using single-line errors should be able to fix it.
```
statement error DataFusion error: Invalid \(non-executable\) plan after Analyzer\ncaused by\nError during planning: Scalar subquery should only return one column, but found 2: t2.t2_id, t2.t2_name
SELECT t1_id, t1_name, t1_int, (select t2_id, t2_name FROM t2 WHERE t2.t2_id = t1.t1_int) FROM t1
```

Reply:
Got it. Fixed. Thanks @jonahgao .